src/script/interpreter.cpp

```diff
@@ -1803,6 +1840,40 @@ uint256 GetOutputsSHA256(const T& txTo)
     return ss.GetSHA256();
 }
 
+// Compute a (single) SHA256 of the concatnation of all outputs
+// with padded 0's to internal fields of nValue, nAsset and nNonce.
+template <class T>
+uint256 GetTapOutputsSHA256(const T& txTo)
+{
+    CHashWriter ss(SER_GETHASH, 0);
+    for (const auto& txout : txTo.vout) {
+        txout.SerializePad(ss);
+    }
+    return ss.GetSHA256();
+}
+
+/** Compute the (single) SHA256 of the concatenation of all asset amounts spent by a tx. */
```

Comment:
"asset amounts" should be "amounts and asset commitments" (similarly for "nonces amounts" below)

Reply:
Changed to "asset and amount commitments" to match the order of serialization